runner/app/pipelines/audio_to_text.py

```diff
@@ -81,7 +93,7 @@ def __init__(self, model_id: str):
             low_cpu_mem_usage=True,
             use_safetensors=True,
             cache_dir=get_model_dir(),
-            attn_implementation="eager",  # TODO: enable flash attention.
+            attn_implementation=attn_implementation,
             **kwargs,
         ).to(torch_device)
 
```

Comment:
There is a runtime warning on startup:
> You are attempting to use Flash Attention 2.0 with a model not initialized on GPU. Make sure to move the model to GPU after initializing it on CPU with `model.to('cuda')`.

Loading the model to the CPU first, and then GPU appeared to clear it
```suggestion
            low_cpu_mem_usage=True,
            use_safetensors=True,
            cache_dir=get_model_dir(),
            **kwargs,
        ).to('cpu')

        # Move the model to the GPU
        model = model.to(torch_device)
        model.config.attn_implementation = attn_implementation
```

Reply:
@eliteprox Thanks for the suggested change. Commited the suggestion :rocket:

Reply:
Strange this issue should not exist if cuda_device is GPU.

Reply:
I think its something with Flash attention in Transformers model as there were some issues answered where this warning disappeared by using the method in the above suggested changes.